http_service/bugbug_http/utils.py

```diff
@@ -5,6 +5,12 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
+import datetime
+import threading
+import time
+from datetime import timedelta
```

Comment:
Nit: `from datetime import datetime, timedelta` so you can use `datetime.now()` instead of `datetime.datetime.now()` in the rest of the file

Reply:
Unfortunatly, if I do as you advise I can no longer patch out the datetime object. See also https://stackoverflow.com/questions/31306080/pytest-monkeypatch-isnt-working-on-imported-function/31746577

Reply:
OK, np!